chart/values.yaml

```diff
@@ -110,6 +110,9 @@ job_memory: "70Mi"
 
 profile_browser_idle_seconds: 60
 
+billing_enabled: false
+sales_email: "sales@webrecorder.org"
```

Comment:
We probably want to default this to empty in the main helm chart. Can change in a follow-up PR.